scaaml/capture/aes/crypto_alg.py

```diff
@@ -72,27 +72,25 @@ def __init__(self,
         # Generate ans save key-text pairs.
         ktp = self._get_new_ktp()
         # Turn what gets generated by ktp into two arrays and save it.
-        keys = []
+        keys_list = []
         texts = []
         while True:
             try:
                 # AcqKeyTextPatternScaaml.new_pair returns (key, text)
                 kt_pair = ktp.new_pair()
-                keys.append(list(kt_pair[0]))
+                keys_list.append(list(kt_pair[0]))
                 texts.append(list(kt_pair[1]))
             except StopIteration:
```

Comment:
For readability, I expect we could rewrite the following code pattern as a simple for loop:
```python
while True:
  try:
     # code
  except StopIteration:
    break
```

Wouldn't the whole `while` loop be equivalent to the following?
```python
for k, t in ktp:
  keys_list.append(k)
  texts.append(t)
```